### DISCUSSION TITLE
Access Dark Mode skin and remove Beautiful Jekyll ad

### DISCUSSION THREAD

**Discussion by efacundoargana** [daattali/beautiful-jekyll#868] [Q&A] [answered]
I've become a Sponsor but I'm not being able to activate dark mode and remove the Beautiful Jekyll ad.

I have not found any information about this beyond the reference that you have to become a sponsor user to access these features.

Does anyone know how to activate this?

**Comment by daattali** [comment, marked as answer]
Please email me dean@attalitech.com and I'll provide you with the steps
